### DISCUSSION TITLE
How to make a static website, where are the generated files?

### DISCUSSION THREAD

**Discussion by glow12121** [datenstrom/community#405] [Ask a question] [answered]
Hello,

I am making a static page in Linux and after the `build` command cannot find the actual site. The manual says it is in the `public` folder - but where is that? I looked in home/Public and searched for any newly created folders - to no avail.

Thank you.

**Comment by glow12121** [comment]
The process itself seems to go without a hitch. The output from PHP:

```
~/Downloads/yellow-master $ php yellow.php build
Your website has been updated: Please run command again
```

**Comment by schulle4u** [comment, marked as answer]
Looks like you haven't completed the last Yellow update. Please run `php yellow.php update` and download remaining files, then try to build your website again.

**Comment by glow12121** [comment]
It works!

Now the problem is the website is built without a theme applied. The theme *is* applied, however, when I `serve` the site. I used the master version, with no additional extensions.

Just in case, terminal output:

> igor@m700 ~ $ cd ~/Downloads/yellow-test
> igor@m700 ~/Downloads/yellow-test $ php yellow.php update
> Your website is up to date
> igor@m700 ~/Downloads/yellow-test $ php yellow.php build
> ERROR building files: Please configure StaticUrl in file 'system/settings/system.ini'!
> Yellow build: 0 files, 1 error
> igor@m700 ~/Downloads/yellow-test $ php yellow.php build
> Yellow build: 18 files, 0 errors
> igor@m700 ~/Downloads/yellow-test $ php yellow.php serve
> Starting built-in web server on http://localhost:8000
> Press Ctrl-C to quit...

**Comment by glow12121** [comment]
Tried this on my other machine (both run Solus Linux) - the same result. The site is `built` with no theme applied but `served` with a theme. The Yellow package I used is just the zip file from your site.

![image](https://user-images.githubusercontent.com/29035180/53954304-e929d780-40e6-11e9-98b1-6a2afddcd37f.png)

**Comment by markseuffert** [comment]
Looks like you're missing one step: Upload the static website to your web server.  [See documentation](
https://github.com/annaesvensson/yellow-generate).

**Comment by glow12121** [comment]
Thanks for pointing this out, Mark!

**Comment by annaesvensson** [comment]
Feel free to close you own issues when they are solved.

**Comment by glow12121** [comment]
Sure, thanks once again. Yellow is great software making the world better.
